arm-logic/2015-08-01-preview/swagger/logic.json

```diff
@@ -3653,8 +3634,24 @@
             "type": "object",
             "properties": {
                 "keyVault": {
-                    "$ref": "#/definitions/ResourceReference",
-                    "description": "The key vault reference."
+                    "type": "object",
```

Comment:
We are using AutoRest composite client here. This resource defintion is already defined in the another spec document which is breaking the code generation. Also, in this document , the resourceReference is being used only once, hence used as inline.